processor/coralogixprocessor/README.md

```diff
@@ -0,0 +1,56 @@
+# Coralogix Processor
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: traces   |
+| Distributions | [contrib] |
+| Warnings      | [Statefulness](#warnings) |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aprocessor%2Fcoralogix%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aprocessor%2Fcoralogix) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aprocessor%2Fcoralogix%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aprocessor%2Fcoralogix) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@crobert-1](https://www.github.com/crobert-1), [@galrose](https://www.github.com/galrose) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+## Description
+
+The Coralogix processor (config name: coralogix) adds attributes to spans that enable features in Coralogix.
+
+## Features
+### DB Statement Blueprints
+This feature enables the processor to create blueprints from sql queries, this means replacing any variables with `?`.
+The blueprint is also hashed to be able to be used with spanmetrics.
+
+The added attributes are `db.statment.blueprint` and `db.statement.blueprint.id`.
+
+If sampling is enabled then it stores the found blueprints in an in-memory cache to be able to send only new blueprints that haven't been seen yet.
+
+If sampling is not enabled it won't cache anything, just add the attributes. 
+
+## Config
```

Comment:
It would be good to include each config option, whether it's required, its default value, and a description.

Reply:
[71a48f9](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33458/commits/71a48f9bc777d75420b43259c42b57756067bebf)